polkadot-parachain/tests/running_the_node_and_interrupt.rs

```diff
@@ -31,24 +32,10 @@ fn running_the_node_works_and_can_be_interrupted() {
 		unistd::Pid,
 	};
 
-	fn run_command_and_kill(signal: Signal) {
-		let _ = fs::remove_dir_all("interrupt_test");
-		let mut cmd = Command::new(cargo_bin("polkadot-parachain"))
-			.args(&["-d", "interrupt_test", "--", "--dev"])
-			.spawn()
-			.unwrap();
-
-		thread::sleep(Duration::from_secs(30));
-		assert!(cmd.try_wait().unwrap().is_none(), "the process should still be running");
-		kill(Pid::from_raw(cmd.id().try_into().unwrap()), signal).unwrap();
-		assert_eq!(
-			common::wait_for(&mut cmd, 30).map(|x| x.success()),
-			Some(true),
-			"the process must exit gracefully after signal {}",
-			signal,
-		);
-	}
-
-	run_command_and_kill(SIGINT);
-	run_command_and_kill(SIGTERM);
+	let base_dir = tempdir().expect("could not create a temp dir");
+
+	let args = &["--", "--dev"];
```

Comment:
@ruseinov 
what about those original args `"-d", "interrupt_test", "--", "--dev"`, they are not needed?

Reply:
Nope, please look at the base_dir logic.